### DISCUSSION TITLE
Change configuration of Controller?

### DISCUSSION THREAD

**Discussion by F0XTA1L** [ndeadly/MissionControl#242] [Q&A]
Is there a possible way that I can configure the plugin so the controller's control setup would be the same as if I had an 8Bitdo dongle plugged into my console?

**Comment by ndeadly** [comment]
There are currently no individual controller configurations, though you can remap the controls for a given controller using the builtin settings to do so in firmwares 10.0.0+

> so the controller's control setup would be the same as if I had an 8Bitdo dongle plugged into my console?

I don't own the 8bitdo dongle so I'm not quite sure what you meant by this

**Reply by F0XTA1L** [reply]
I tried this earlier today but couldn't configure the controls to the way the 8bit configured them

**Comment by impeeza** [comment]
Hi, you are mixing two concepts:

When you use the 8BitDo Dongle, the console think you did connect a USB Switch Pro Controller, the dongle make all the conversion between Bluetooth and USB connections.

Mission control is a system service who take over the Bluetooth stack of the Switch and allows the switch "think" what "any" bluetooth controller is a original Switch Pro controller connected via Bluetooth.

so there is two different things.

**Reply by F0XTA1L** [reply]
Hi yes I know they are 2 different concepts I just wanted to configure the controls to the same layout as the dongle.

**Comment by impeeza** [comment]
Ah, ok.  You can use the Nintendo switch menu to remap the buttons as the switch see the controller as a original Nintendo switch pro controller

**Reply by F0XTA1L** [reply]
Tried that but doesn't seem to pick up a few buttons which is weird.

**Reply by ndeadly** [reply]
Are you using an Xbox controller by any chance? If so, you need to update the controller to the latest firmware for the correct button mappings

**Reply by F0XTA1L** [reply]
Alright I am using an xbox one how should I update it?

**Reply by ndeadly** [reply]
Either through an Xbox One console, or through the Xbox Accessories app on windows. See the [FAQ](https://github.com/ndeadly/MissionControl#frequently-asked-questions) for more details

**Reply by F0XTA1L** [reply]
Thanks alot that Fixed it now it's excacly the same layout as the 8bit was! Works Like a charm!
